main.py

```diff
@@ -1,10 +1,55 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as f:
+        file_json = json.load(f)
+        for key, value in file_json.items():
```

Comment:
Reread checklist, you dont need to write the who function body inside your context manager with open. Also use more descriptive names than key and value.